zstacklib/zstacklib/gpu/vendors/tianshu.py

```
@@ -0,0 +1,56 @@
+            pci_address = cls.normalize_pci_address(parts[0].strip())
+                
+            gpu_infos.append(GPUInfo(
+                pci_address=pci_address,
+                memory=parts[1].strip(),
+                power=parts[2].strip(),
+                serial_number=parts[3].strip()
+            ))
+        return gpu_infos
+
+    @classmethod
+    def get_metric_cmd(cls, is_windows=False):
```

Ruff (0.14.13)

Unused class method argument: `is_windows`

(ARG003)